src/segments/python.go

```diff
@@ -22,6 +22,7 @@ const (
 	FetchVirtualEnv      properties.Property = "fetch_virtual_env"
 	UsePythonVersionFile properties.Property = "use_python_version_file"
 	FolderNameFallback   properties.Property = "folder_name_fallback"
+	FallbackNames        properties.Property = "fallback_names"
```

Comment:
```suggestion
	DefaultVenvNames        properties.Property = "default_venv_names"
```

Reply:
I also changed the doc, test and schema to follow the new name